app/views/candidate_interface/continuous_applications/course_choices/blocked_submissions/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1 class="govuk-heading-l">Submission blocked</h1>
```

Comment:
This content is used in multiple places. Can't we have it in one partial?

Reply:
The other is the 2023 content that we don't show anymore and will be deleted in my cleanup PR part 2, so I think it is best to leave for deletion.